src/features/common/Layout/RedeemPopup/index.tsx

```diff
@@ -4,6 +4,7 @@ import styles from './RedeemPopup.module.scss';
 import i18next from '../../../../../i18n';
 import tenantConfig from '../../../../../tenant.config';
 import { UserPropsContext } from '../UserPropsContext';
+import { useRouter } from 'next/router';
```

Comment:
@prachigarg19 It may be better to conditionally render `CookiePolicy` and `RedeemPopup` within `/src/features/common/Layout/index.tsx` instead of adding this logic within both `CookiePolicy` and `RedeemPopup` as it keeps the code cleaner and avoids duplication.